src/blocks.js

```diff
@@ -3092,6 +3092,17 @@ BlockMorph.prototype.userMenu = function () {
         }
         return menu;
     }
+    if (contains(
+        ['doBroadcast', 'doBroadcastAndWait', 'receiveMessage'],
```

Comment:
what about 'doSend', shouldn't that also be part of this list?

Reply:
Totally. Adding it now.